pexec/process.go

```diff
@@ -29,11 +29,14 @@ type ProcessConfig struct {
 	OneShot bool
 	// Optional. When present, we will try to look up the Uid of the named user
 	// and run the process as that user.
-	Username    string
-	Log         bool
-	LogWriter   io.Writer
-	StopSignal  syscall.Signal
-	StopTimeout time.Duration
+	Username string
+	// Additional environment variables to pass through to the process.
+	// Will overwrite exising environment variables.
+	EnvironmentVariables map[string]string
```

Comment:
Could shorten to `Env`. Opinions?

Reply:
I'm more worried about the JSON struct tag being `environment_variables`; maybe we could change that to `environment`? And this field to `Environment`? My opinion is not very strong, though.

Reply:
Ah just saw in your scope that you're using `"env"` as the JSON field for module env vars. If you're still planning on doing that, I think we should use `"env"` for processes, too (i.e. JSON struct tag of the field below). Whether the name of the go struct field is `Env` or `Environment` does not seem particularly important to me.

Reply:
Swapped to `Environment`